DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/TopCourses/TopCoursesController.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Centre.TopCourses

```

Comment:
If this controller is only going to be used for the page that is linked from the centre dashboard card, then it should probably go in the Dashboard folder like the RankingController and ContractDetailsController